log/log.go

```diff
@@ -30,18 +32,80 @@ var levelPrefix = [...]string{
 	LevelFatal:    "[FATAL] ",
 }
 
+var levelPriority = [...]syslog.Priority{
+	LevelDebug:    syslog.LOG_DEBUG,
+	LevelInfo:     syslog.LOG_INFO,
+	LevelWarning:  syslog.LOG_WARNING,
+	LevelError:    syslog.LOG_ERR,
+	LevelCritical: syslog.LOG_CRIT,
+	LevelFatal:    syslog.LOG_EMERG,
+}
+
 // Level stores the current logging level.
-var Level = LevelDebug
+var Level = LevelInfo
+
+var (
+	// SysLoggers maps each logging level to a *log.Logger that will be used for it.
+	SysLoggers = make([]*golog.Logger, LevelFatal+1)
```

Comment:
These variables should probably not be exported, since you don't expect people to be able to set or read them externally, right?

For Boulder's needs, the nicest thing would be to call some exported method, e.g. `log.UseSyslog(writer)` to enable syslogging. I think that would imply, instead of the multiple syslogger instances created below, using a single syslog instance with a big switch:

```
switch level {
  case syslog.LOG_DEBUG:
    writer.Debug(message)
  ...
```